src/clients/Governance.ts

```diff
@@ -666,4 +667,26 @@ export class Governance extends API {
     )
     return response.data
   }
+
+  async getUserNotifications(address: string) {
+    const response = await this.fetch<ApiResponse<Notification[]>>(
+      `/notifications/user/${address}`,
+      this.options().method('GET')
+    )
+    return response.data
+  }
+
+  async sendNotification(recipient: string, title: string, body: string, type: number, url: string) {
+    const response = await this.fetch<ApiResponse<string>>(
+      `/notifications/send`,
+      this.options().method('POST').authorization({ sign: true }).json({
+        recipient,
+        title,
+        body,
+        type,
+        cta: url,
```

Comment:
You could call this parameter url, since in the route it is doing

```
return await NotificationService.sendNotification({
    type,
    title,
    body,
    recipient,
    url: cta,
    customType: NotificationCustomType.Announcement,
  })
  ```